src/core/PathUtil.cpp

```diff
@@ -60,7 +60,7 @@ namespace lmms::PathUtil
 				if (error) { *error = (!s || projectPath.isEmpty()); }
 				break;
 			}
-			default                   : return QString("");
+			default                     : return QString("");
```

Comment:
This also looks like an accidental change, please undo this as well.

```suggestion
			default                   : return QString("");
```